### ISSUE TITLE
typechain is incompatible with non returned value message

### ISSUE THREAD

**Issue by shunsukew** [Brushfam/typechain-polkadot#19]
typechain-polkadot generated types and codes are incompatible with non returned value messages.
When messages have non returned value (precisely it returns `()` empty tuple) like below,
```
#[ink(message)]
pub fn flip(&mut self) {
    self.value = !self.value;
}
```
Program will face with error,
```
  1) flipper test
       Can flip the state:
     Error: the object {
  "issue": "OUTPUT_IS_NULL"
} was thrown, throw an Error :)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)

```

I will add procedures to reproduce once it is ready.

**Comment by shunsukew** [comment]
Metadata that works
```
"messages": [
        {
          "args": [],
          "docs": [
            " A message that can be called on instantiated contracts.",
            " This one flips the value of the stored `bool` from `true`",
            " to `false` and vice versa."
          ],
          "label": "flip",
          "mutates": true,
          "payable": false,
          "returnType": {
            "displayName": [
              "bool"
            ],
            "type": 0
          },
          "selector": "0x633aa551"
}
]
```

Metadata that doesn't work
```
"messages": [
        {
          "args": [],
          "docs": [
            " A message that can be called on instantiated contracts.",
            " This one flips the value of the stored `bool` from `true`",
            " to `false` and vice versa."
          ],
          "label": "flip",
          "mutates": true,
          "payable": false,
          "returnType": null,
          "selector": "0x633aa551"
        }
]
```

**Comment by coreggon11** [comment]
@shunsukew thanks for opening this issue, we will address it soon

**Comment by varex83** [comment]
Closed in #99
